ejerciciosProg/personasEnElCine.ts

```diff
@@ -20,35 +21,34 @@ function asignarBoolean(): boolean {
 }
 
 function cargaButacas(arrayButaca: boolean[]) {
-  let longitudArray: number = arrayButaca.length - 1;
-  for (let elemento: number = 0; elemento <= longitudArray; elemento++) {
-    arrayButaca[elemento] = asignarBoolean();
+  let longitudArray: number = arrayButaca.length;
+  for (let indice: number = 0; indice < longitudArray; indice++) {
+    arrayButaca[indice] = asignarBoolean();
   }
 }

```

Comment:
- Cabiamos el nombre **elemento** por **indice**. Por convención se suele usar _casi siempre_ la variable **indice** para los for.
- No es necesario restarle 1 a la longitud del array, porque entonces la variable **longitudArray** no estaría representando la longitud del array. En este caso lo que hacemos es que el for vaya de **0** hasta un valor menor a la longitud del array, por eso cambiamos `elemento <= longitudArray` por `indice < longitudArray`.